README.md

```diff
@@ -320,7 +328,7 @@ RESTful React also exports a `Poll` component that will poll a backend endpoint
 ```jsx
 import { Poll } from "restful-react"
 
-<Poll path="/deployLogs" resolve={data => data && data.data}>
+<Poll path="deployLogs" resolve={data => data && data.data}>
```

Comment:
I removed the leading `/`s here but I am not sure the code will support this change. See https://github.com/contiamo/restful-react/pull/33#issuecomment-417657956

Reply:
I am not sure that Poll should support the nested relative route structure. It could be huge resource hog when nesting multiple. thoughts? @peterszerzo @TejasQ @fabien0102

Reply:
Polls would never be nested, but should have a similar API to the other components in order to not confuse users.

Reply:
Just to be clear, you are talking specifically about the leading slashes of the `path`? @TejasQ

Reply:
Yes.